source/Minable.cpp

```diff
@@ -43,25 +43,27 @@ void Minable::Load(const DataNode &node)
 
 	for(const DataNode &child : node)
 	{
-		if(child.Token(0) == "display name" && child.Size() >= 2)
+		const string &key = child.Token(0);
+		bool hasValue = child.Size() >= 2;
+
+		if(key == "display name" && hasValue)
 			displayName = child.Token(1);
-		else if(child.Token(0) == "noun" && child.Size() >= 2)
+		else if(key == "noun" && hasValue)
 			noun = child.Token(1);
 		// A full sprite definition (frame rate, etc.) is not needed, because
 		// the frame rate will be set randomly and it will always be looping.
-		else if(child.Token(0) == "sprite" && child.Size() >= 2)
+		else if(key == "sprite" && hasValue)
 			SetSprite(SpriteSet::Get(child.Token(1)));
-		else if(child.Token(0) == "hull" && child.Size() >= 2)
+		else if(key == "hull" && hasValue)
 			hull = child.Value(1);
-		else if(child.Token(0) == "random hull" && child.Size() >= 2)
+		else if(key == "random hull" && hasValue)
 			randomHull = max(0., child.Value(1));
-		else if((child.Token(0) == "payload" || child.Token(0) == "explode") && child.Size() >= 2)
+		else if(key == "payload" && hasValue)
+			payload.emplace_back(child);
+		else if(key == "explode" && hasValue)
```

Comment:
ik this isnt the goal of the PR but whilst we're here shouldnt we check hasvalue once at the start of the if, given all of them require it?